app/views/privacy_policies/show.html.erb

```diff
@@ -0,0 +1,103 @@
+<h1 class="govuk-heading-l">How we look after your personal data</h1>
+  <h2 class="govuk-heading-m">Department for Education (DfE)</h2>
+    <p class="govuk-body">Mentions of &#8220;us&#8221; and &#8220;we&#8221; mean DfE and &#8220;you&#8221; means anyone using this service.</p>
```

Comment:
Can we not use special characters? They really don't make things easy to read :/

Reply:
Yeah, I got bored of that quite quickly...